src/Resources/Resources/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Make -Scope optional in *-AzPolicyAssignment cmdlets
```

Comment:
>*-AzPolicyAssignment cmdlets [](start = 26, length = 28)

add a note that this will default to the context's subscription scope if not provided